lucid/modelzoo/vision_base.py

```diff
@@ -87,21 +87,15 @@ def _layers_from_list_of_dicts(model_class, list_of_dicts):
   return tuple(layers)
 
 
-class ModelPropertiesMetaClass(type):
-
-  @property
-  def name(cls):
-      return cls.__name__
-
-
-class Model(with_metaclass(ModelPropertiesMetaClass, object)):
+class Model():
```

Comment:
you should remove the `with_metaclass` import from the top of this file

Reply:
removed!